### PR TITLE
RELEASE: Share a Case

### PR DESCRIPTION
**Pull request by shaunganley**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json [hunk 1]
```diff
@@ -1,2 +0,0 @@
-[
-]
```
Comment (johnoneill46) [inline definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json:2]: Thought the plan was to keep these empty non-prod files around so we don't have large PRs with unnecessary changes and tests being constantly updated each time. Doesn't matter either way but just something to consider

**Review by johnoneill46 — APPROVED** [review]

File: definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json (hunk 1 above)
Comment (shaunganley) [reply definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json:2]: we shouldn't use plain nonprod files anymore as we can't track these against an epic so I'm removing to avoid things being accidentally added in here

Comment (johnoneill46) [reply definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json:2]: Ah good shout 👍  Let's make sure to keep an eye on it going forward then in future reviews

**Review by yarpo — APPROVED** [review]
